src/components/meetup.js

```diff
@@ -0,0 +1,37 @@
+import React, { Fragment, useState, useEffect } from "react"
+
+const MeetUp = () => {
+  const [events, setEvents] = useState([])
+
+  const corsAPI = "https://cors-anywhere.herokuapp.com/"
+  const meetupURL =
+    "https://www.meetup.com/Syracuse-Software-Development-Meetup/events/json/"
+
+  useEffect(() => {
+    fetch(corsAPI + meetupURL, {
```

Comment:
It mihgt be a good idea to url encode `meetupURL` in this situation: `corsAPI + encodeURIComponent(meetupUrl)`. I'm not 100% sure on this, it's just a hunch and in practice it may not matter.